contracts/BosonRouter.sol

```diff
@@ -179,6 +199,28 @@ contract BosonRouter is
         emit LogOrderCreated(tokenIdSupply, msg.sender, metadata[5], ETHETH);
     }
 
+     /**
+     * @notice Internal function called by other TKNTKN requestCreateOrder functions to decrease code duplication. 
+     * Price and deposits are specified in tokens.
+     * @param _tokenPriceAddress address of the token to be used for the price
+     * @param _tokenDepositAddress address of the token to be used for the deposits
+     * @param _tokensSent total number of tokens sent. Must be equal to seller deposit * quantity
+     * @param deadline deadline after which permit signature is no longer valide. See EIP-2612
```

Comment:
"valide" -> "valid".
Also repeats below.

Reply:
Included in next commit.